inst/WORDLIST

```diff
@@ -1,24 +1,25 @@
 Biomarker
-Forkers
-Hoffmann
-MAEs
-Reproducibility
-TLG
-UI
-UIs
-UX
 cloneable
 customizable
 favicon
 favicons
+Forkers
 funder
+Hoffmann
+JSON
+MAEs
 omics
 pre
 programmatically
 repo
+Reproducibility
 reproducibility
```

Comment:
Uppercase and lower case are both necessary?

Reply:
this is how this WORDLIST work : / it's stupid